Resources/Prototypes/_NF/Catalog/Fills/Items/weapon_cases_shuttle_guns.yml

```
@@ -167,6 +167,29 @@
       - id: WeaponShotgunSawnNonlethal
       - id: BoxBeanbag
 
+# WeaponSVT
+- type: entity  #made By VividPups added in with PR 1770
+  parent: WeaponCaseLongShuttle
+  id: ShuttleWeaponSVT
+  noSpawn: true
+  components:
+  - type: StorageFill
+    contents:
+      - id: WeaponSVT
+      - id: MagazineSVT
+
+# WeaponSVTNonlethal # Frontier - TODO: Restore this later
+# - type: entity
+  # parent: WeaponCaseLongShuttle
+  # id: ShuttleWeaponSVTNonlethal
+  # noSpawn: true
+  # components:
+  # - type: StorageFill
+    # contents:
+      # - id: N/A
```

Make this as painless a replacement as possible.
```suggestion
      # - id: WeaponSVTNonlethal
```